Projects/hCoreUI/Sources/Views/InfoCard.swift

```diff
@@ -226,49 +175,6 @@ extension View {
 }
 
 public enum InfoCardLayoutStyle {
-    case roundedRectangle
-    case rectange
-}
-
-struct InfoCardStyle: ViewModifier {
-    let type: InfoCardType
-    @Environment(\.hInfoCardLayoutStyle) var layoutStyle
-    func body(content: Content) -> some View {
-        switch layoutStyle {
-        case .rectange:
-            content
-                .background(
-                    Rectangle()
-                        .fill(getBackgroundColor)
-                        .overlay(
-                            Rectangle()
-                                .strokeBorder(hBorderColor.primary, lineWidth: 0.5)
-                        )
-                )
-        case .roundedRectangle:
-            content
-                .background(
-                    Squircle.default()
-                        .fill(getBackgroundColor)
-                        .overlay(
-                            RoundedRectangle(cornerRadius: .cornerRadiusL)
-                                .strokeBorder(hBorderColor.primary, lineWidth: 0.5)
-                        )
-                )
-        }
-    }
-
-    @hColorBuilder
-    var getBackgroundColor: some hColor {
-        switch type {
-        case .info:
-            hSignalColor.Blue.fill
-        case .attention:
-            hSignalColor.Amber.fill
-        case .error:
-            hSignalColor.Red.fill
-        case .campaign:
-            hSignalColor.Green.fill
-        }
-    }
+    case defaultStyle
```

Comment:
can we just use default and banner ?
I know Xcode doesn't allow you to name the property default but if you do `default` instead it should work
Its because is default is key word, but if you put `` its ignoring it